man/org.freedesktop.systemd1.xml

```diff
@@ -1492,6 +1502,11 @@ node /org/freedesktop/systemd1 {
       boot loader or initrd implementation. In these cases the respective pairs of timestamps are both 0,
       indicating that no data is available.</para>
 
+      <para><varname>UnitsLoadTimestamp</varname> encode <constant>CLOCK_REALTIME</constant>
```

Comment:
`<varname>UnitsLoadTimestamp</varname> and <varname>UnitsLoadTimestampMontonic</varname> encode …`